packages/build/tamagui-build.js

```diff
@@ -485,7 +485,7 @@ async function esbuildWriteIfChanged(
     allowOverwrite: true,
     keepNames: false,
     sourcemap: true,
-    sourcesContent: false,
+    sourcesContent: true,
```

Comment:
To let the debugger get the source code content to display the original code.

[esbuild document](https://esbuild.github.io/api/#sources-content):

> The `sourcesContent` field is an optional field that contains all of the original source code. This is helpful for debugging because it means the original source code will be available in the debugger.

Reply:
Update: this is not actually needed for VSCode debugger to get the source code.

What actually matters is the `//# sourceMappingURL=` comment in the output `.mjs` file.